packages/web-extension/package.json

```diff
@@ -1,6 +1,6 @@
 {
-  "name": "carbon-devtools",
-  "version": "2.7.7",
+  "name": "carbon-devtools-v10",
```

Comment:
Do you mean to set the extension's display name as well? Or does this string get built into the display name? 
<img width="627" alt="image" src="https://github.com/user-attachments/assets/c4667036-1ae9-4029-bdf4-eb13c1db0ca2">

Reply:
I didn't see anywhere to modify a "display name". I'm wondering if there's not a way to change the extension name.

Reply:
Per https://developer.chrome.com/docs/extensions/reference/manifest/name, the name is set in the manifest.

https://github.com/carbon-design-system/devtools/blob/base/packages/web-extension/src/manifest.json#L2

That name is set during the webpack build.

https://github.com/carbon-design-system/devtools/blob/base/packages/web-extension/webpack.common.js#L166

I'm not suggesting that the name should be hardcoded or set build-time. Just sharing where it's happening.

Reply:
Thanks for the pointer on this, I've just confirmed that building the package (`yarn build`) does alter the manifest.json file's name. Updates incoming.